src/Compilers/CSharp/Test/Symbol/Symbols/Source/FileModifierTests.cs

```diff
@@ -2074,6 +2075,28 @@ class E
             Diagnostic(ErrorCode.ERR_FileTypeDisallowedInSignature, "C").WithArguments("C", "E.M<T>(T)").WithLocation(10, 30));
     }
 
+    [Fact, WorkItem(62435, "https://github.com/dotnet/roslyn/issues/62435")]
+    public void Constraints_02()
```

Comment:
Consider also covering type constraint on local function.
Consider covering all occurrences of type constraints (classes, structs, interfaces, records, delegate types) by parameterizing this test.